### ISSUE TITLE
RC6 Forms: when value is null, control is not disabled

### ISSUE THREAD

**Issue by Krisa** [angular/angular#11253]
**I'm submitting a bug**  (check one with "x")
```
[x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
```

**Current behavior**
```
form = new FormGroup({
   'first': new FormControl({value: null, disabled: true}, Validators.required),
   'last': new FormControl('Drew', Validators.required)
})
```

the control `first` will not be disabled

**Expected/desired behavior**
`first` shall be disabled because `disabled` is true, even so the input is null. If the value is empty (''), it will not be disabled either. In other words, the input seems disabled only, and only if the input has a `value` defined

**Reproduction of the problem**

**What is the expected behavior?**

**What is the motivation / use case for changing the behavior?**

**Please tell us about your environment:**

* **Angular version:** 2.0.0-rc.6
 
* **Browser:** [ Chrome 52 ] 
 
* **Language:** [ TypeScript ]

**Comment by kara** [comment]
@Krisa It's working fine for me: http://plnkr.co/edit/Yid3Cgxyiyu2s741FuQm?p=preview.  Can you post a plunker that reproduces?

**Comment by Krisa** [comment]
Yes you're right @kara, I've just found the issue. My specific example is correct to reproduce the issue, but the difference with yours, is that actually the `Validators.required` sets back the input as editable. If you add that to your example, then it will not be disabled. 

Probably, this does not make sense to set an input required... disabled. But the example in my app is a bit more complex than that, because the form is set dynamically by querying a dabatase server and this field shall be required and potentially disabled based on the value of the form... This brings back to what we discussed just before here https://github.com/angular/angular/pull/10994
